nss-day-cohort-30/bangazon-site-suave-salmon
# unsaved changes fixed

**Fixes:**
Fixed unsaved data

**Changes proposed in this request & reasons behind organizational or architectural decisions:**

Details page now displays date created, description, title, price, quantity, city, and product type. The edit and index hyperlinks have been removed. Working details title links can also be found on the products page. 

'Category' page now displays all product types 

Steps to test:
git fetch --all
checkout to SecondTicketMo
git pull origin SecondTicketMo
Run program
Naviagate to Products/Details/*id of item you want to see*

Additionally, navigate to /Products
All titles should be hyperlinks that go to the details view

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):

none


Link to feature ticket:
-https://github.com/nss-day-cohort-30/bangazon-site-suave-salmon/issues/1
@suave-salmon
